react/react1/week2/Todo/my-app/src/ToDoList.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import ToDo from "./ToDo";
+
+const ToDoList = ({ toDoList, handleToggle, handleFilter }) => {
+  return (
+    <div>
+      {toDoList.map((todo) => {
+        return (
+          <ToDo
+            todo={todo}
+            handleToggle={handleToggle}
+            handleFilter={handleFilter}
```

Comment:
When you are not using the arguments `handleToggle `and `handleFilter `in the ToDo component, Why are you passing them?